src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php

```diff
@@ -218,15 +211,16 @@ public function create(array $options = [])
         $product->setEnabled($options['enabled']);
         $product->setMainTaxon($options['main_taxon']);
         $product->setArchetype($options['product_archetype']);
-        $product->setShippingCategory($options['shipping_category']);
```

Comment:
For now, as we do not support shipping categories and I am not sure yet if we will - bringing this back when we properly support them will be easy.